index.js

```diff
@@ -42,10 +42,11 @@ export function autocomplete ({
   // build query
   const query = {
     api_key: apiKey,
-    sources,
     text
   }
 
+  if (sources !== null && sources.length > 0) query.sources = sources
```

Comment:
Could this snag if sources is `undefined`?  I would maybe just change the conditional to `if (sources && sources.length > 0)`.